common/ideologies/00_leader_ideologies.txt

```diff
@@ -1015,6 +1015,84 @@ ideology_royalist = {
 	}
 }
 
+ideology_kaiserist = {
+	icon = "gfx/interface/icons/ideology_icons/ideology_leader/ideology_leader_royalist.dds"
+	
+	character_ideology = yes
+
+	lawgroup_governance_principles = {
+		law_monarchy = strongly_approve
+		law_presidential_republic = strongly_disapprove
+		law_parliamentary_republic = strongly_disapprove
+		law_theocracy = approve
+		law_council_republic = strongly_disapprove
+	}
+
+	lawgroup_citizenship = {
+		law_ethnostate = strongly_disapprove
+		law_national_supremacy = approve
+		law_racial_segregation = strongly_approve
+		law_cultural_exclusion = approve
+		law_multicultural = strongly_disapprove	
+	}
+
+	lawgroup_church_and_state = {
+		law_state_religion = strongly_approve
+		law_freedom_of_conscience = disapprove
+		law_total_separation = strongly_disapprove
+	}
+
+	lawgroup_distribution_of_power = {
+		law_universal_suffrage = strongly_disapprove
+		law_census_voting = strongly_disapprove
+		law_wealth_voting = disapprove
+		law_landed_voting = approve
+		law_anarchy = strongly_disapprove
+		law_oligarchy = approve
+		law_autocracy = approve		
+	}
+	
+	possible = {
```

Comment:
Should it have a section for has to be bohemia? as far as I can tell like this it appears no matter if it is bohemia or another nation. Additionally, should be a negative if they are a republic in addition to the negative for theocracy

Reply:
Unless there’s some background code that automatically changes/spawns in the ideology from the list, it shouldn’t appear from any thing.

Reply:
Isn't that the leader_weight section?
If it isn't it is probably good to disregard

Reply:
No, its the possible = {  stuff that allows it to spawn, so if its left blank I believe it should never appear.